src/command_default.rs

```diff
@@ -20,6 +21,14 @@ pub fn run_command_default(channel: String) -> Result<()> {
     save_config_db(&config_data)
         .with_context(|| "`default` command failed to save configuration db.")?;
 
+    if std::env::consts::OS != "windows" {
+        match config_data.installed_channels.get(&channel).unwrap() {
+            JuliaupConfigChannel::SystemChannel { version } =>
+                create_symlink(&version, &"julia".to_string())?,
+            _ => eprintln!("Symlink creation currently not supported for linked channels."),
```

Comment:
Is there a reason for that?

Reply:
Since linked channels are specified by a system command as well as command line arguments, we can't just create a symlink, we would probably want to create a small shell script that just calls that command and passes on any additional arguments.